rasa/core/channels/twilio.py

```diff
@@ -128,9 +136,13 @@ async def health(_: Request) -> HTTPResponse:
         async def message(request: Request) -> HTTPResponse:
             sender = request.form.get("From", None)
             text = request.form.get("Body", None)
-
             out_channel = self.get_output_channel()
 
+            if self._is_location_message(request):
+                text = "/locationData{{'Latitude': {lat},'Longitude': {long}}}".format(
+                    lat=request.form.get("Latitude"), long=request.form.get("Longitude")
+                )
```

Comment:
Would you know if it is possible that the variable `text` is not None at this point and gets overwritten?

Reply:
Ok so this what i receive when i send a location message
![image](https://user-images.githubusercontent.com/46184494/229291772-a8bd4182-2a71-4ff7-9bec-e0bab8e4a3f3.png)
as you can see there is no "Body" key value so no text is sent when sending a location message only label. Where as when i send normal text.
![image](https://user-images.githubusercontent.com/46184494/229291850-e5f93b55-b251-42bc-8b97-760e1dc0db38.png)
i see a "Body" key value here.

what i can i do is add a check of if text or body is none before overriding but i saw that twilio never sends a text in a body when sending location media

Reply:
what do you recommend ?

Reply:
@m-mohsin-ali Can you please add a comment that text is always `None` with a location message from Twilio?